CHANGELOG.md

```diff
@@ -18,6 +18,8 @@
 - Add a `Cis4Contract` for interacting with Cis4 contracts.
 - Add a new `web3id` module that contains types and functionality for
   construcing Web3ID credentials and verifying Web3ID proofs.
+- Add a `commission_rates` field to `CurrentPaydayBakerPoolStatus` which yields the commission rates 
+  of the baker for the reward period (requires a node with version > 6.0).
```

Comment:
Version 3 of the SDK has been released. You need to move this changelog up into "Unreleased changes" section.